docs/examples/use_cases/pytorch/resnet50/main.py

```diff
@@ -461,14 +519,14 @@ def adjust_learning_rate(optimizer, epoch, step, len_epoch):
     if epoch >= 80:
         factor = factor + 1
 
-    lr = args.lr * (0.1 ** factor)
+    lr = args.lr*(0.1**factor)
 
     """Warmup"""
     if epoch < 5:
-        lr = lr * float(1 + step + epoch * len_epoch) / (5. * len_epoch)
+        lr = lr*float(1 + step + epoch*len_epoch)/(5.*len_epoch)
 
-    if(args.local_rank == 0 and step % args.print_freq == 0 and step > 1):
-        print("Epoch = {}, step = {}, lr = {}".format(epoch, step, lr))
+    # if(args.local_rank == 0):
+    #     print("epoch = {}, step = {}, lr = {}".format(epoch, step, lr))
```

Comment:
Why is this commented out?

Reply:
It is also in https://github.com/NVIDIA/apex/blob/master/examples/imagenet/main_amp.py. Remvoed.